### ISSUE TITLE
[weird bug] 9 pokemon spots occupied with no pokemon

### ISSUE THREAD

**Issue by PokemasterMike** [NECROBOTIO/NecroBot#2465]
if i log in with my phone, it says i have 18/250 pokemon even though there are only 9 in my inventory (that i can see). happened on every account if used for couple hours+ on necrobot (both google and ptc)

anybody got the same thing ??

**Comment by emwedk** [comment]
change this to false 
  "TransferWeakPokemon": false,

**Comment by PokemasterMike** [comment]
what ? did you even read my post ??? (the bot also says i have 9 pokemon (the same ones i can see on my phone))

**Comment by emwedk** [comment]
do u have a timed resetter on you bots?
cuz the update from 5.0 to 6.0 added this, and it just starts transfering all pokemons beneath the set CP and/or IV

**Comment by emwedk** [comment]
im sure i read it, now im not so sure if i understood it Kappa

**Comment by PokemasterMike** [comment]
@emwedk are you replying to the wrong post ? this has nothign to do with IV and/or transfer

**Comment by emwedk** [comment]
u say you see 18 in the bot but only got 9 in inv?

**Comment by Rykee** [comment]
It shows the correct amount for me. Could you give us a screenshot?

**Comment by theloser** [comment]
That's not a bot issue I think but a general error with Pokemon Go: the number of pokemon you see in your Pokemon Go app on your phone is the number of pokemon **plus** the number of eggs you own (although there is a second tab which reads "9 of 9 eggs"). I already reported this to Niantic weeks ago and I got no response. And I think because the bot receives this number from Niantic it is wrong, too.
